specs/datapackages/README.md

```diff
@@ -0,0 +1,130 @@
+# CKAN / Data Package integration
+
+## Overview
+[Data Packages](http://data.okfn.org/doc/data-package) are a really simple specification for packaging up data files to allow easy reuse and sharing. They are essentially a descriptor file named `datapackage.json` which contains metadata about the dataset and points towards the data files.   It has a focus on simplicity and extensibility.
+
+For instance, there is a special flavour of Data Package aimed at [Tabular data](http://data.okfn.org/doc/tabular-data-package) which enforces the use of the [JSON Table schema](http://dataprotocols.org/json-table-schema/), another spec part of the same family of standards which allows to describe the structure of CSV files.
+
+The full Data Package spec can be found here:
+
+http://dataprotocols.org/data-packages/
+
+There is a growing number of [tools](http://data.okfn.org/tools) being developed around Data Packages. Improving its support within CKAN would allow out of the box integration with this ecosystem of tools.
+
+Also besides direct support for handling Data Packages, there is a number of areas where adopting and implementing standards around them could help deal with long standing issues in CKAN. Perhaps the most clear example is the JSON Table Schema, which would allow publishers to properly describe data shown on previews or being imported to the DataStore. Implementing this would be a massive win in order to build better import and ETL tools into CKAN (or even just to get the previews right).
+
+## Prior work
+
+Open Knowledge developed a while ago http://datapackager.okfn.org, a tool for creating Data Packages from data files.
+
+It was implemented in CKAN (as a CKAN [extension](https://github.com/ckan/ckanext-datapackager) that heavily customized some parts of the CKAN UI).
+
+It contains functions to translate Data Packages metadata into CKAN datasets and viceversa, but perhaps more interestingly includes logic functions and an interface for modifying the schema for tabular resources:
+
+![Schemas](http://i.imgur.com/i5L5oNz.png)
+
+
+
+## Main goals
+
+1. Allow importing Data Packages into CKAN, creating the necessary datasets and resources, both via the UI and the API
+
+2. Allow exporting CKAN datasets as Data Packages, regardless of whether they were imported from Data Packages in a first instance or not.
+
+3. Support schema definition for CKAN resources, allowing publishers to describe the contents of published files
+
+
+For achieving this it seems sensible to reuse as much stuff as we can from ckanext-datapackager, extending it if necessary
+
+The first two points could be implemented either in core or separate extension, but I'd like to propose that the third regarding resource schemas is implemented in core (in a generic form). It seems central enough to all future work around data import, cleaning, etc (eg closer integration with DataStore) to justify it.
```

Comment:
s/The first two points could be implemented either in core or separate extension/The first two points should be implemented as a core extension/